stories/modules-tagmanager-components.stories.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * Tag Manager Module Component Stories.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { storiesOf } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import { WithTestRegistry } from '../tests/js/utils';
+import { STORE_NAME } from '../assets/js/modules/tagmanager/datastore';
+import { STORE_NAME as CORE_SITE, AMP_MODE_PRIMARY } from '../assets/js/googlesitekit/datastore/site/constants';
+import * as fixtures from '../assets/js/modules/tagmanager/datastore/__fixtures__';
+import AccountSelect from '../assets/js/modules/tagmanager/components/common/AccountSelect';
+import WebContainerSelect from '../assets/js/modules/tagmanager/components/common/WebContainerSelect';
+import AMPContainerSelect from '../assets/js/modules/tagmanager/components/common/AMPContainerSelect';
+import UseSnippetSwitch from '../assets/js/modules/tagmanager/components/common/UseSnippetSwitch';
+import ExistingTagNotice from '../assets/js/modules/tagmanager/components/common/ExistingTagNotice';
+import ExistingTagError from '../assets/js/modules/tagmanager/components/common/ExistingTagError';
+import ErrorNotice from '../assets/js/modules/tagmanager/components/common/ErrorNotice';
+import AccountCreate from '../assets/js/modules/tagmanager/components/common/AccountCreate';
+
+storiesOf( 'Tag Manager Module/Components', module )
+	.add( 'AccountSelect', () => {
```

Comment:
Let's wrap all these components in some setup `div`s like we do in e.g. Analytics - it provides a more representative preview (mainly just because of the white background).